index.html

```diff
@@ -84,7 +93,7 @@ <h3>Multi-Post Stories</h3>
             <button type="button" class="project-button">See Project</button>
```

Comment:
- [ ] **Contrast ratio ("color contrast")**:
The page should have a minimum contrast ratio of at least 4.5:1 for normal-size text. You can go ahead and adjust the colors to pass this test

Reply:
Hello @Georjane 
Thank you for the suggestions to help me improve my code base, however, I don`t understand the changes required by you above because I thought I did the same thing in my PR.